data/xml/Q20.xml

```diff
@@ -0,0 +1,171 @@
+      <pages>1–18</pages>
+      <url hash="c153b336">Q20-1001</url>
+    </paper>
+    <paper id="2">
+      <title>AMR-To-Text Generation with Graph Transformer</title>
+      <author><first>Tianming</first><last>Wang</last></author>
+      <author><first>Xiaojun</first><last>Wan</last></author>
+      <author><first>Hanqi</first><last>Jin</last></author>
+      <doi>10.1162/tacl_a_00297</doi>
+      <abstract>Abstract meaning representation (AMR)-to-text generation is the challenging task of generating natural language texts from AMR graphs, where nodes represent concepts and edges denote relations. The current state-of-the-art methods use graph-to-sequence models; however, they still cannot significantly outperform the previous sequence-to-sequence models or statistical approaches. In this paper, we propose a novel graph-to-sequence model (Graph Transformer) to address this task. The model directly encodes the AMR graphs and learns the node representations. A pairwise interaction function is used for computing the semantic relations between the concepts. Moreover, attention mechanisms are used for aggregating the information from the incoming and outgoing neighbors, which help the model to capture the semantic information effectively. Our model outperforms the state-of-the-art neural approach by 1.5 BLEU points on LDC2015E86 and 4.8 BLEU points on LDC2017T10 and achieves new state-of-the-art performances.</abstract>
+      <pages>19–33</pages>
+      <url hash="9dce31b7">Q20-1002</url>
+    </paper>
+    <paper id="3">
+      <title>What BERT Is Not: Lessons from a New Suite of Psycholinguistic Diagnostics for Language Models</title>
+      <author><first>Allyson</first><last>Ettinger</last></author>
+      <doi>10.1162/tacl_a_00298</doi>
+      <abstract>Pre-training by language modeling has become a popular and successful approach to NLP tasks, but we have yet to understand exactly what linguistic capacities these pre-training processes confer upon models. In this paper we introduce a suite of diagnostics drawn from human language experiments, which allow us to ask targeted questions about information used by language models for generating predictions in context. As a case study, we apply these diagnostics to the popular BERT model, finding that it can generally distinguish good from bad completions involving shared category or role reversal, albeit with less sensitivity than humans, and it robustly retrieves noun hypernyms, but it struggles with challenging inference and role-based event prediction— and, in particular, it shows clear insensitivity to the contextual impacts of negation.</abstract>
+      <pages>34–48</pages>
+      <url hash="50cfa1c2">Q20-1003</url>
+    </paper>
+    <paper id="4">
+      <title>Membership Inference Attacks on Sequence-to-Sequence Models: Is My Data In Your Machine Translation System?</title>
```

Comment:
manual fixed-case for "Is"

Reply:
I don't think fixed-casing gets applied to this; see many instances in P19.xml for example.

Reply:
My intuition is that if the title contains a question sentence before or after a colon, it's clearly a standalone utterance (a different speech act) and thus the part after the colon should be capitalized.

Thus IMO, here are how such bib entries should be rendered:

Membership inference attacks on sequence-to-sequence models: Is my data in your machine translation system?
Is my data in your machine translation system? Membership inference attacks on sequence-to-sequence models
Is my data in your machine translation system?: Membership inference attacks on sequence-to-sequence models
"Is my data in your machine translation system?" Membership inference attacks on sequence-to-sequence models
"My data is in your machine translation system": Membership inference attacks on sequence-to-sequence models

However, it couldn't be done completely manually, because we could have questions as NP modifiers (with or without quotation marks), e.g.

Natural language understanding of What? questions

I don't have a strong opinion about sentence-capitalizing the part after the colon if neither side is a question/exclamation/quotation (each is an NP, VP, or declarative sentence). Probably simplest not to add any capitalization, thus

My data is in your machine translation system: membership inference attacks on sequence-to-sequence models
Finding data in your machine translation system: membership inference attacks on sequence-to-sequence models
Hostile intrusions: membership inference attacks on sequence-to-sequence models
MemInfSeq2Seq: membership inference attacks on sequence-to-sequence models

etc.